README.md

```diff
@@ -40,4 +40,9 @@ To contribute:
 8. Run this code for upgrade the database after the migrate
     ```bash
     flask db upgrade
-    ```
\ No newline at end of file
+    ```
+
+9. Run this code for formatting
+    ```bash
+    yapf -i [path/file].py
```

Comment:
Does it mean we need to install this package first?

Should it be added in the requirements.txt file?

Reply:
yep, we already did, we haven't written it yet in README.md

https://github.com/project-zusammen/ioc-backend/blob/master/requirements.%20txt#L25